### ISSUE TITLE
Move Exercism student tools to this repo

### ISSUE THREAD

**Issue by macta** [exercism/pharo#17]
Move the spike on gitlab:macta/exercism over to this repo, and update the readme instructions so that we have all of the exercises and tooling in one place. This means that exercism devs (not students) can easily contribute to the tooling as well.

**Comment by macta** [comment]
@kytrinyx I am still confused how we can test our progress so far - pharo still doesn't appear as a track for the exercism client.

**Comment by samWson** [comment]
> pharo still doesn't appear as a track for the exercism client.

@macta I'll do some digging on this part for you. I'll see if this is something for `exercism/discussions`, `exercism/cli`, or something else entirely.

**Comment by macta** [comment]
That would be helpful - I popped something on the request-new-language repo and got a vague reply that there is no beta concept anymore. This needs chasing as surely they want people to test there tracks before launch? I found it quite deflating and it tempered my enthusiasm. You really want the joy of making some small changes and seeing how it works, rinse and repeat...

**Comment by kytrinyx** [comment]
> I popped something on the request-new-language repo and got a vague reply that there is no beta concept anymore.

That was not an official reply. We do have a way to make tracks available for maintainers during development, and will be expanding any necessary features to be able to beta test it (I think this should work already, but it's been a scramble in the past weeks, so I'm not sure of the status).

For the pharo track, what happened was that we imported it and it was breaking the site (our fault, not yours), and we didn't have time to fix the import script to handle it.

I've opened an issue to sort out the naming of the exercise directory so that it will integrate with the site and our tooling--once that is done I can add it back to get imported into the live site.

I'm sorry about the confusion.

I will be away for the next 5 days, but will check back on August 1st first thing to see what we can do to help get this sorted out properly.
